tests/core/test_playback.py

```diff
@@ -59,24 +59,24 @@ def translate_uri(self, uri):
 # playbackprovider Since we rely on our DummyAudio to actually emit events we
 # need a "real" backend and not a mock so the right calls make it through to
 # audio.
-class TestBackend(pykka.ThreadingActor, backend.Backend):
+class MyTestBackend(pykka.ThreadingActor, backend.Backend):
     uri_schemes = ['dummy']
 
     def __init__(self, config, audio):
-        super(TestBackend, self).__init__()
-        self.playback = TestPlaybackProvider(audio=audio, backend=self)
+        super(MyTestBackend, self).__init__()
+        self.playback = MyTestPlaybackProvider(audio=audio, backend=self)
 
 
-class BaseTest(unittest.TestCase):
+class BaseTest(object):
     config = {'core': {'max_tracklist_length': 10000}}
     tracks = [Track(uri='dummy:a', length=1234),
               Track(uri='dummy:b', length=1234),
               Track(uri='dummy:c', length=1234)]
 
-    def setUp(self):  # noqa: N802
+    def setup_method(self, method):  # noqa: N802
```

Comment:
You can remove the `# noqa: N802` comment, which is to ignore the camel-casing in `setUp`.